app/src/main/java/org/fossasia/pslab/fragment/ControlFragmentAdvanced.java

```diff
@@ -69,25 +73,240 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         CheckBox checkBoxControlAdvanced3 = (CheckBox) view.findViewById(R.id.checkbox_control_advanced3);
         CheckBox checkBoxControlAdvanced4 = (CheckBox) view.findViewById(R.id.checkbox_control_advanced4);
 
-        etWidgetControlAdvanced1.init(getContext(), 1.0, 10.0, 5000.0);
-        etWidgetControlAdvanced2.init(getContext(), 1.0, 10.0, 5000.0);
-        etWidgetControlAdvanced3.init(getContext(), 1.0, 10.0, 5000.0);
-        etWidgetControlAdvanced4.init(getContext(), 0.1, 0.0, 1.0);
-        etWidgetControlAdvanced5.init(getContext(), 1.0, 0.0, 360.0);
-        etWidgetControlAdvanced6.init(getContext(), 0.1, 0.0, 1.0);
-        etWidgetControlAdvanced7.init(getContext(), 1.0, 0.0, 360.0);
-        etWidgetControlAdvanced8.init(getContext(), 0.1, 0.0, 1.0);
-        etWidgetControlAdvanced9.init(getContext(), 1.0, 0.0, 360.0);
-        etWidgetControlAdvanced10.init(getContext(), 0.1, 0.0, 1.0);
-        etWidgetControlAdvanced11.init(getContext(), 1.0, 0.0, 360.0);
+        etWidgetControlAdvanced1.setInputType(InputType.TYPE_NULL);
+        etWidgetControlAdvanced2.setInputType(InputType.TYPE_NULL);
+        etWidgetControlAdvanced3.setInputType(InputType.TYPE_NULL);
+        etWidgetControlAdvanced4.setInputType(InputType.TYPE_NULL);
+        etWidgetControlAdvanced5.setInputType(InputType.TYPE_NULL);
+        etWidgetControlAdvanced6.setInputType(InputType.TYPE_NULL);
+        etWidgetControlAdvanced7.setInputType(InputType.TYPE_NULL);
+        etWidgetControlAdvanced8.setInputType(InputType.TYPE_NULL);
+        etWidgetControlAdvanced9.setInputType(InputType.TYPE_NULL);
+        etWidgetControlAdvanced10.setInputType(InputType.TYPE_NULL);
+        etWidgetControlAdvanced11.setInputType(InputType.TYPE_NULL);
+
+        etWidgetControlAdvanced1.setText("10.0");
+        etWidgetControlAdvanced2.setText("10.0");
+        etWidgetControlAdvanced3.setText("10.0");
+        etWidgetControlAdvanced4.setText("0.0");
+        etWidgetControlAdvanced5.setText("0.0");
+        etWidgetControlAdvanced6.setText("0.0");
+        etWidgetControlAdvanced7.setText("0.0");
+        etWidgetControlAdvanced8.setText("0.0");
+        etWidgetControlAdvanced9.setText("0.0");
+        etWidgetControlAdvanced10.setText("0.0");
+        etWidgetControlAdvanced11.setText("0.0");
+
+
+        etWidgetControlAdvanced1.setOnClickListener(new View.OnClickListener() {
+
```

Comment:
Reformat the code a little bit to remove these extra blank lines! :)

Reply:
Android Studio has a built in functionality for reformatting code. If you don't find it go to `Help` -> `Find Actions` -> type `Reformat Code` . Do this for every file you edit so that the alignment is taken care of.

Reply:
I will do it asap